src/main/java/com/airlines/dao/DbConnector.java

```diff
@@ -0,0 +1,36 @@
+package com.airlines.dao;
+
+import com.zaxxer.hikari.HikariDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ResourceBundle;
+
+public class DbConnector {
+    HikariDataSource datasource;
+
+    public DbConnector() {
+        ResourceBundle resourceBundle = ResourceBundle.getBundle("db");
+        String url = resourceBundle.getString("url");
+        String username = resourceBundle.getString("username");
+        String password = resourceBundle.getString("password");
+
+        datasource = new HikariDataSource();
+        datasource.setJdbcUrl(url);
+        datasource.setUsername(username);
+        datasource.setPassword(password);
+    }
+
+    public Connection getDBConnection() {
+
+        Connection dbConnection = null;
+        try {
+            dbConnection = datasource.getConnection();
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+        return dbConnection;
```

Comment:
no need to return there.
Just move it to try ..block:
try {
  return dataSource.getConnection();

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/3d7c7d3d96fe1b8bd3698ee76a32a9f3d5b5354e